frontend/src/components/AnalyticsDrawer/CensusAnalytics.tsx

```diff
@@ -9,11 +9,7 @@ export let CensusAnalytics = withStore(
   'serviceAreas'
 )(({ store }) => {
 
-  let selectedServiceAreas = store.get('serviceAreas')
-  if (store.get('selectedServiceAreas')) {
-    selectedServiceAreas = store.get('selectedServiceAreas')!
-  }
-
+  let selectedServiceAreas = store.get('selectedServiceAreas') ? store.get('selectedServiceAreas')! : store.get('serviceAreas')
```

Comment:
is this exclamation point necessary? the first part of the ternary operator is already checking if it's null

Reply:
It's complaining :(